app/services/stripe_service/api/payments.rb

```diff
@@ -107,15 +113,28 @@ def payout(tx)
 
         seller_gets = payment[:subtotal] - payment[:commission]
 
-        case stripe_api.destination(tx[:community_id])
-        when :platform
+        case stripe_api.charges_mode(tx[:community_id])
+        when :separate
           seller_gets -= payment[:real_fee] || 0
           if seller_gets > 0
-            result = stripe_api.perform_transfer(tx[:community_id], seller_account[:stripe_seller_id], seller_gets.cents, payment[:sum].currency, payment[:subtotal].cents, payment[:stripe_charge_id])
+            result = stripe_api.perform_transfer(tx[:community_id],
+              seller_account[:stripe_seller_id],
+              seller_gets.cents,
+              payment[:sum].currency,
+              payment[:subtotal].cents,
+              payment[:stripe_charge_id],
+              {order_id: tx[:id]}
+            )
```

Comment:
The stripe API methods tend to have quite a few arguments. How about switching to keyword arguments so that it's clear what is what?